application-github-statistics-ui/src/main/resources/GitHubStats/CommittersMacro.xml

```diff
@@ -20,75 +20,228 @@
  * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
 -->
 
-<xwikidoc>
+<xwikidoc version="1.3" reference="GitHubStats.CommittersMacro" locale="">
```

Comment:
Well I need a ssx and it adds an object that uses the new format. So a solution would be to put the changes manually. I am not trilled about it.